evap/evaluation/models.py

```diff
@@ -550,10 +559,14 @@ def general_contribution_has_questionnaires(self):
 
     @property
     def all_contributions_have_questionnaires(self):
-        return (
-            self.general_contribution
-            and not self.contributions.annotate(Count("questionnaires")).filter(questionnaires__count=0).exists()
-        )
```

Comment:
There is a test (now failing) asserting that we check for the existence of the general_contribution and return False here if it does not yet exist.

Boils down to the broader discussion about how to handle general contributions in #1839